scrnaseq.Rmd

```diff
@@ -635,7 +638,7 @@ genes_highestExpr = lapply(sc_names, function(i) {
   unlist() %>%
   unique()
 
-genes_highestExpr_counts = purrr::map_dfc(sc[sc_names], .f=function(s) s[["RNA"]][["counts_median"]][genes_highestExpr,]) 
+genes_highestExpr_counts = purrr::map_dfc(sc[sc_names], .f=function(s) s[["RNA"]][["counts_median"]][genes_highestExpr, ]) 
```

Comment:
Either please remove the `.f=` or add it to all `purrr` functions.

Reply:
This is nothing that has to do with the PR =) ... (this has been there before)

I remember that without `.f` I got a complaint in R. How would you write it?

Reply:
`genes_highestExpr_counts = purrr::map_dfc(sc[sc_names], function(s) return(s[["RNA"]][["counts_median"]][genes_highestExpr, ]))`

Reply:
Removed it